Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -961,6 +962,9 @@ static String mutationObserverNotificationString()
 
     determineCurrentlySnappedPage();
 
+    // FIXME: Make the overlay scroll with the tiles instead of repainting constantly.
+    m_frame->page()->findController().didInvalidateFindRects();
```

Comment:
`protectedPage()`

Reply:
Good point! This does not match all the others.